sdk/include/xdk/winnt_old.h

```diff
@@ -952,8 +952,8 @@
 #define DLL_PROCESS_ATTACH    1
 #define DLL_THREAD_ATTACH    2
 #define DLL_THREAD_DETACH    3
-#ifdef __WINESRC__
-#define DLL_WINE_PREATTACH    8 /* Never called, but defined for compatibility with Wine source */
+#ifndef __REACTOS__ /* We don't use DLL_WINE_PREATTACH any more */
+#define DLL_WINE_PREATTACH    8
 #endif
```

Comment:
Ideally, this (documenting) definition should be moved to some header in `/sdk/include/reactos/wine`:
maybe `port.h` or `winternl.h` (as a fallback) or create an `unused.h` or ...

Reply:
Moved to `sdk/include/reactos/port.h`.

Reply:
Almost: I meant in `wine/` subdirectory.

Reply:
Moved.